features/step_definitions/node.rb

```diff
@@ -683,3 +683,8 @@
    admin.cli_exec(:oadm_cordon_node, opts)
   end
 end
+
+Given /^I store the schedulable workers without taints in the#{OPT_SYM} clipboard$/ do |cb_name|
+  step %Q{I store the schedulable workers in the :#{cb_name} clipboard}
```

Comment:
@ropatil010 when using `OPT_SYM`, the variable means the user can put in an optional symbol name, so you need set a default clipboard name is none is defined.  You can do so by  ||
```
cb_name ||= :nodes
```
After doing that, you can call the step like so 
```
I store the schedulable workers without taints in the clipboard
```

Reply:
Fixed, Can you PTAL and help to review. 
https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/ocp-common/job/Runner/1024429/console